test/support/utils.h

```diff
@@ -754,6 +755,18 @@ struct MaxAbsFunctor<::std::complex<_Tp>>
     }
 };
 
+struct tuple_add_fn
```

Comment:
```suggestion
struct TupleAddFunctor
```
...to match the style of other functors here, obviously update the usages as well.

Reply:
I have made this change.